readmestaging20180818

```diff
@@ -0,0 +1 @@
+Creating new branch staging20180818
```

Comment:
cleanup

Reply:
oh yikes!

Reply:
removed -- it got in here because I merged WFP's staging into my branch